Gopkg.lock

```diff
@@ -0,0 +1,93 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
Check that this will not be problemtatic when used on projects that use govendor